packages/react/use-query-param/src/hooks/useQueryParam.spec.ts

```diff
@@ -1,16 +1,16 @@
 import { renderHook, waitFor } from '@testing-library/react';
 import { mocked } from 'jest-mock';
 import Router, { NextRouter, useRouter } from 'next/router';
-import { useQueryParam } from './useQueryParam';
 import { renderWithSuspense } from '../utils';
+import { useQueryParam } from './useQueryParam';
 
 jest.mock('next/router');
 
 const mockRouter = mocked(Router);
 const mockUseRouter = mocked(useRouter as () => Partial<NextRouter>);
 
 describe('useQueryParam', () => {
-  it('첫 번째 파라미터와 일치하는 이름의 쿼리 파라미터를 읽을 수 있다.', () => {
+  it('can read the query parameter with a name matching the first parameter.', () => {
```

Comment:
While implementing the function, I also changed the description of the test code from Korean to English.